InstantGrinder/InstantGrinder/InstantGrinderCommandModule.cs

```diff
@@ -121,12 +121,19 @@ public sealed class InstantGrinderCommandModule : CommandModule
             Plugin.GridGridGroup((MyPlayer) player, gridGroup);
 
             Context.Respond($"Finished grinding: \"{gridName}\"", Color.White);
-
             var playerInventory = (MyInventory) player.Character.GetInventory();
-            if (playerInventory.CurrentMass > playerInventory.MaxMass ||
-                playerInventory.CurrentVolume > playerInventory.MaxVolume)
+            if ((playerInventory.CurrentMass > playerInventory.MaxMass ||
+                playerInventory.CurrentVolume > playerInventory.MaxVolume) && !Plugin.GridInventoryIsEmpty)
```

Comment:
`GridInventoryIsEmpty` shouldn't be a global state. Maybe you can return it from `GrindGridGroup()` as a bool or wrapped in a "report" struct. Why? Because the property *can* get in an invalid state in case the command failed to initialize by accident -> affect the next command.

Reply:
Hi, i will merge all your new changes,  and try to rethink how to update the inventory, as this is the only thing that broken.